content/docs/user-guide/contributing/docs.md

```diff
@@ -53,11 +53,45 @@ We will review your PR as soon as possible. Thank you for contributing!
 
 ## Development environment
 
-We highly recommend running this web app locally to check documentation or blog
-changes before submitting them, and it's quite necessary when making changes to
-the website engine itself. Source code and content files need to be properly
+We highly recommend running this web app to check documentation or blog changes
+before submitting them, and it's quite necessary when making changes to the
+website engine itself. Source code and content files need to be properly
 formatted and linted as well, which is also ensured by the full setup below.
 
+> ⚡ You can create a dev env online using Gitpod (a fully featured online IDE),
+> it'll automatically clone the repo, install the dependencies, and run the app
+> (takes a few minutes), so that you can start straight away.
+>
+> <a href="https://gitpod.io/#https://github.com/iterative/dvc.org">
+>   <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open in Gitpod">
+> </a>
+
+<details>
+
+### How to make a Pull Request with Gitpod?
+
+- Fork the repo by selecting Find Command from View in the top menu and then
+  type fork and press enter:
+
+  ![image](https://user-images.githubusercontent.com/46004116/76392493-75727880-6393-11ea-8939-41e6c36477de.png)
```

Comment:
We should host these images on the dvc.org (e.g. to process them properly with sharp)? not sure what are the consequences of sharing it like this. Btw, @nisarhassan12 how did you create them?

Reply:
Maybe they're somewhere in this repo: https://github.com/gitpod-io/workspace-images

Reply:
Ah no, I think he pasted them into a Github conversation like this:

![image](https://user-images.githubusercontent.com/1477535/81460584-e2c04100-916b-11ea-86b6-6f36b0317bac.png)
> `![image](https://user-images.githubusercontent.com/1477535/81460584-e2c04100-916b-11ea-86b6-6f36b0317bac.png)`

Reply:
Which is interesting. I wonder it Github ever deletes images from it's CDN.

Reply:
yeah! True I pasted them in Github comment. Sorry, this is a bad idea.